Examples/Io/Csv/include/ActsExamples/Io/Csv/CsvSimHitReader.hpp

```diff
@@ -0,0 +1,67 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2019 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#pragma once
+
+#include "Acts/Geometry/GeometryIdentifier.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "ActsExamples/Framework/IReader.hpp"
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+namespace Acts {
+class Surface;
+}
+
+namespace ActsExamples {
+
+/// Read in a simhit collection in comma-separated-value format.
+///
+/// This reads one files per event in the configured input directory. By default
+/// it reads files in the current working directory. Files are assumed to be
+/// named using the following schema
+///
+///     event000000001-<stem>.csv
+///     event000000002-<stem>.csv
+///
+/// and each line in the file corresponds to one simhit.
+class CsvSimHitReader final : public IReader {
+ public:
+  struct Config {
+    /// Where to read input files from.
+    std::string inputDir;
+    /// Input filename stem.
+    std::string inputStem;
+    /// Output simulated (truth) hits collection.
+    std::string outputSimulatedHits;
```

Comment:
```suggestion
    std::string outputSimHits;
```
For consistency with other algorithms.